### ISSUE TITLE
Error: ENOENT: no such file or directory /public/static/d/8652886.json'

### ISSUE THREAD

**Issue by iassainov** [pristas-peter/gatsby-plugin-graphql-component#3]
Hi @pristas-peter ,

First of all, amazing plugin! It's about to help me big time!

I've just cloned the example repo, and tried to build it, but it throws the following error:
```
 ERROR #11321  PLUGIN

"gatsby-plugin-graphql-component" threw an error while running the onPreBuild lifecycle:

ENOENT: no such file or directory, open '/Users/ilyas_assainov/Desktop/Projects/Shipa/public/static/d/8652886.json'



  Error: ENOENT: no such file or directory, open '/Users/ilyas_assainov/Desktop/Projects/Shipa/public/static/d/8652886.json'

not finished Generating image thumbnails - 6.425s

(sharp:56268): GLib-CRITICAL **: 22:31:48.308: g_hash_table_lookup: assertion 'hash_table != NULL' failed

(sharp:56268): GLib-CRITICAL **: 22:31:48.690: g_hash_table_lookup: assertion 'hash_table != NULL' failed
```

I've noticed there is no such path in the public folder, the path is probably `/Users/ilyas_assainov/Desktop/Projects/Shipa/public/page-data/sq/d/8652886.json` - that's where that file is located!

Could you please fix it? Or suggest a fix so I can make a PR?

**Comment by iassainov** [comment]
I just saw there was a [merged commit](https://github.com/pristas-peter/gatsby-plugin-graphql-component/commit/dc8da3dede677f3f023e7df9cd3148c879c10641) by @bebjakub , that fixed it, but the npm package is outdated. Could you please update it?

**Comment by pristas-peter** [comment]
@bebjakub @iassainov Sorry for the late response. Published as 0.2.4.
